### PR TITLE
[ENH]customized lower tooltip. updated reflection.md

### PR DESCRIPTION
**Pull request by sreejithmunthikodu**
- Customized lower tooltip
- Updated reflection.md
- Couldn't remove tooltip for hex bins

### CODE REVIEW

File: reflection.md [hunk 1]
```diff
@@ -1,50 +1,72 @@
...
+- Our movie selection app is successfully built-in plotly/dashR and deployed using Heroku
+- The app is simple and easy to use 
+- The plots are interactive. Selecting a bar in the upper plot would highlight the same movie in the lower plot
+- The overplotting issue we had in the Python version of the app was successfully resolved by choosing a suitable plot
+- We were able to accomplish what we proposed in milestone 3, with some minor modifications based on the feedback we received  
 
 ## Limitations
 
-- Our app does not have the interactivity between the two plots that we proposed. This is partly due to the time constraints for the project, and based on our discussion with the course instructor.  
+- Our app does not have the interactivity between the two plots that we proposed. This is partly due to the time constraints for the project and based on our discussion with the course instructor.
```
Comment (techrah) [inline reflection.md]: We have this functionality now

Comment (techrah) [reply reflection.md]: But maybe mention that it's not fully working as expected. For example, when new data is loaded, we can't reset the lower chart.

**Review by techrah — APPROVED** [review]

File: reflection.md (hunk 1 above)
Comment (sreejithmunthikodu) [reply reflection.md]: Thanks for pointing it. I removed it. The limitations of the interactivity are already mentioned.

Comment (techrah) [reply reflection.md]: nice. thanks.

**Pull request merged**